docs/svo_fps/svo_fps.rst

```diff
@@ -0,0 +1,170 @@
+.. doctest-skip-all
+
+.. _astroquery.svo_fps:
+
+**********************************************************
+SVO Filter Profile Service Queries (`astroquery.svo_fps`)
+**********************************************************
+
+Getting started
+===============
+
+This is a python interface for querying the Spanish Virtual Observatory's
+Filter Profile Service (`SVO FPS <http://svo2.cab.inta-csic.es/theory/fps/>`_).
+It allows to fetch filter data (index, transmission data, filter list, etc.)
+from the service, as astropy tables.
+
+Get index (master list) of all Filters
+--------------------------------------
+
+The filter index (all available filters with their properties) can be listed as:
+
+.. code-block:: python
+
+    >>> from astroquery.svo_fps import SvoFps
+    >>> index = SvoFps.get_filter_index()
+    >>> index.info
+    <Table masked=True length=5139>
+            name          dtype  unit
+    -------------------- ------- ----
+    FilterProfileService  object     
+                filterID  object     
+          WavelengthUnit  object     
+           WavelengthUCD  object     
+              PhotSystem  object     
+            DetectorType  object     
+                    Band  object     
+              Instrument  object     
+                Facility  object     
+        ProfileReference  object     
+    CalibrationReference  object     
+             Description  object     
+                Comments  object     
+          WavelengthMean float32   AA
+           WavelengthEff float32   AA
+           WavelengthMin float32   AA
+           WavelengthMax float32   AA
+                WidthEff float32   AA
+           WavelengthCen float32   AA
+         WavelengthPivot float32   AA
+          WavelengthPeak float32   AA
+          WavelengthPhot float32   AA
+                    FWHM float32   AA
+               PhotCalID  object     
+                  MagSys  object     
+               ZeroPoint float32   Jy
+           ZeroPointUnit  object     
+                    Mag0 float32     
+           ZeroPointType  object     
+               AsinhSoft float32     
+        TrasmissionCurve  object     
+
+
+Get transmission data for a specific Filter
+-------------------------------------------
+
+If you know the filterID of the filter, for which you wish to retrieve
```

Comment:
Mention that you can get the filter ID from either of the other two query outputs.